ServiceDocumentSampleComponents/serviceDocumentSampleComponents/force-app/main/default/lwc/bodyText/bodyText.js-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>57.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__ServiceDocument</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__ServiceDocument">
+            <supportedFormFactors>
+                <supportedFormFactor type="Large" />
+            </supportedFormFactors>
+            <property name="text" type="string" default="Body text" />
+        </targetConfig>
+    </targetConfigs>
+</LightningComponentBundle>
```

Comment:
I don't think it's super necessary to include this sample cmp given that the rich text cmp can achieve the same thing.